tools/rosnode/src/rosnode/__init__.py

```diff
@@ -62,6 +63,8 @@
 
 NAME='rosnode'
 ID = '/rosnode'
+# Line is defined at clients/rospy/src/rospy/impl/tcpros_base.py at line 477
```

Comment:
Please do not refer to a line number since that is going to change if the file is being edited in the future. Instead refer to the file and a symbol, e.g. `The string is defined in clients/rospy/src/rospy/impl/tcpros_base.py TCPROSTransport.get_transport_info`.